src/components/design/Edit.tsx

```diff
@@ -0,0 +1,61 @@
+import 'semantic-ui-css/semantic.min.css';
+import Form                         from '@rjsf/semantic-ui';
+import { Button }                   from '@trussworks/react-uswds';
+import { kebabCase }                from 'lodash';
+import { IPageData, IQuestionData } from '../../survey/IStepData';
+import { DesignLayout }             from '../wizard/DesignLayout';
+import { getStepSchema }            from '../../schema/edit';
```

Comment:
`src/schema/edit.ts` vs `src/components/design/Edit.tsx` is a bit confusing, but I guess it's ok since I can't think of any better naming.